frontend/src/components/pages/club_display/NumClubsToDisplay.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import { DEFAULT_NUM_DISPLAYED } from "../club_display/club_display_consts";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    formControl: {
+      margin: theme.spacing(1),
+      marginBottom: theme.spacing(2),
+      marginRight: theme.spacing(8),
+      minWidth: 120,
+    },
+  }),
+);
+
+interface NumClubsToDisplayProps {
+  handleNumClubsChange(event: React.ChangeEvent<HTMLInputElement>): void;
+  numClubsDisplayed: number;
+  showClubsMemberOf: boolean;
```

Comment:
```suggestion
  handleNumClubsChange(event: React.ChangeEvent<HTMLInputElement>): void,
  numClubsDisplayed: number,
  showClubsMemberOf: boolean,
```
Switch semicolons to commas on interface.